docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ax = perf_df.plot.bar(color='#7400ff', ylim=(1,550), rot=0, label="UDFs", ylabel="Speedup factor")
</pre><p> </p><p>Can the <code>ylabel</code> be  <code>UDF Kind</code>  and the yticks be <code>Numeric</code>,<code>String</code> and <code>Grouped</code>?</p><p> </p><p>Some commentary at the beginning of the section can clarify what these mean.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Yup, added some comments